module.go

```diff
@@ -40,6 +40,7 @@ type NewModuleArgs struct {
 	BPFObjPath      string
 	BPFObjBuff      []byte
 	SkipMemlockBump bool
+	KernelLogLevel  C.uint
```

Comment:
```suggestion
	KernelLogLevel  uint32
```

Use go types here and do the conversion inside the wrappers.